src/containers/task-group.js

```diff
@@ -0,0 +1,90 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+import Task from '../components/task';
+
+class TaskGroup extends Component {
+
+  processTaskList(list, givenMonth, givenType) {
+    let resultsArray = [];
+    const loop = list.map( (item) => {
+      if(item.month === givenMonth && item.type === givenType) {
+        resultsArray.push(item);
+      };
+    });
+    if(resultsArray.length) {
```

Comment:
Probably easier to just do `return !!resultsArray.length` (and get true/false, rather than true/undefined).

That said (again), I take back my filter comment; this entire method can just be `return list.some(item => item.month === givenMonth && item.type === givenType)`.